admin/build_targets/centos-7/Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN ["yum", "install", "--assumeyes", "git", "ruby-devel", "python-devel", "pyth
 # See https://github.com/jordansissel/fpm/issues/611
 RUN ["gem", "install", "fpm"]
 
-RUN ["pip", "install", "twisted==15.0.0", "characteristic==14.1.0", "virtualenv==12.0.5", "effect==0.1a13" , "boto==2.30.0", "requests==2.4.3", "requests-file==1.0", "ipaddr==2.1.11", "eliot==0.6.0"]
+RUN ["pip", "install", "twisted==15.0.0", "characteristic==14.1.0", "virtualenv==12.0.5", "effect==0.1a13" , "boto==2.30.0", "requests==2.4.3", "requests-file==1.0", "ipaddr==2.1.11", "eliot==0.6.0", "pyrsistent == 0.9.1"]
```

Comment:
I couldn't see why Pyrsistent is needed in the package build environment? 

TBH I can't remember why I added all these explicit Python packages to the build environment, couldn't we just create a virtualenv and install the Flocker[release] requirements and run the build-package-entrypoint script inside it?